### PR TITLE
Mimics fullscreen when fullscreen api is not supported

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

* Mimics fullscreen when fullscreen api is not supported by browser/webview

### CODE REVIEW

**Comment by christianmemije** [comment]
Rebased.

File: kolibri/plugins/document_pdf_render/assets/src/views/index.vue [hunk 1]
```diff
@@ -1,47 +1,64 @@
 <template>
 
-  <div ref="container" class="container" allowfullscreen>
```
Comment (rtibbles) [inline kolibri/plugins/document_pdf_render/assets/src/views/index.vue:3]: Something seems to have gone very awry here - the pdf renderer index.vue appears to be an exact copy of the HTML5 renderer index.vue now - is this intentional?

Comment (christianmemije) [reply kolibri/plugins/document_pdf_render/assets/src/views/index.vue:3]: Definitely not. :man_facepalming:

Comment (christianmemije) [reply kolibri/plugins/document_pdf_render/assets/src/views/index.vue:3]: @rtibbles Fixed issue.

**Pull request merged**